pages/linux/uvcdynctrl.md

```diff
@@ -0,0 +1,31 @@
+# uvcdynctrl
+
+> A libwebcam command line tool to manage dynamic controls in uvcvideo.
+
+- List all available cameras:
+
+`uvcdynctrl -l`
+
+- Specify the device to use (defaults to `video0`):
+
+`uvcdynctrl -d {{device_name}}`
+
+- List available controls:
+
+`uvcdynctrl -c`
+
+- Set a new control value (for negative values: `-s '{{control_name}}' -- -42`):
```

Comment:
Are you saying that negative values need to be prefixed with a -- ? That seems odd.

Reply:
Yep! That's how the negative argument works. Looking at the command help, ubuntu man page and uvcdynctrl source code confirms this.

Reply:
Thanks for clarifying !

Reply:
In that case, change the description to - (for negative values, add `--` before `{{value}}`)